README.md

```diff
@@ -1,13 +1,11 @@
 ![PcapPlusPlus Logo](https://pcapplusplus.github.io/img/logo/logo_color.png)
 
-[![GitHub Actions](https://github.com/seladb/PcapPlusPlus/workflows/Build%20and%20test/badge.svg)](https://github.com/seladb/PcapPlusPlus/actions?query=workflow%3A%22Build+and+test%22)
-[![Build Status](https://api.cirrus-ci.com/github/seladb/PcapPlusPlus.svg)](https://cirrus-ci.com/github/seladb/PcapPlusPlus)
-[![Build status](https://ci.appveyor.com/api/projects/status/4u5ui21ibbevkstc/branch/master?svg=true)](https://ci.appveyor.com/project/seladb/pcapplusplus/branch/master)
-[![CodeQL status](https://github.com/seladb/PcapPlusPlus/workflows/CodeQL/badge.svg)](https://github.com/seladb/PcapPlusPlus/actions?query=workflow%3A%22CodeQL%22)
-[![Language grade: C/C++](https://img.shields.io/lgtm/grade/cpp/g/seladb/PcapPlusPlus.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/seladb/PcapPlusPlus/context:cpp)
-<a href="https://twitter.com/intent/follow?screen_name=seladb">
-    <img src="https://img.shields.io/twitter/follow/seladb.svg?label=Follow%20PcapPlusPlus" alt="Follow PcapPlusPlus" />
-</a>
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/seladb/PcapPlusPlus/Build%20and%20test?logo=github&style=flat)](https://github.com/seladb/PcapPlusPlus/actions?query=workflow%3A%22Build+and+test%22)
+[![Cirrus CI - Base Branch Build Status](https://img.shields.io/cirrus/github/seladb/PcapPlusPlus?logo=cirrusci&style=flat)](https://cirrus-ci.com/github/seladb/PcapPlusPlus)
+[![AppVeyor](https://img.shields.io/appveyor/build/seladb/PcapPlusPlus/master?logo=appveyor&style=flat)](https://ci.appveyor.com/project/seladb/pcapplusplus/branch/master)
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/seladb/PcapPlusPlus/CodeQL?label=CodeQL&logo=github&style=flat)](https://github.com/seladb/PcapPlusPlus/actions?query=workflow%3A%22CodeQL%22)
+[![Twitter Follow](https://img.shields.io/twitter/follow/seladb?label=Follow%20PcapPlusPlus&style=social)](https://twitter.com/intent/follow?screen_name=seladb)
+![GitHub Repo stars](https://img.shields.io/github/stars/seladb/PcapPlusPlus?style=social)

```

Comment:
I really like it, thank you! 🙏 

Here are a few additional suggestions:
- Maybe we can override the label in each badge to specify the CI system instead of a generic "build" label, for example: https://img.shields.io/cirrus/github/seladb/PcapPlusPlus?label=Cirrus%20CI&logo=cirrusci
- For some reason the AppVeyor logo is blue by default, maybe we can override it to white? https://img.shields.io/appveyor/build/seladb/PcapPlusPlus/master?label=AppVeyor&logo=appveyor&logoColor=white
- It's funny that the stars badge refers to 1950 stars as 1.9k, whereas GitHub refers to it as 2k. I don't think it's that bad, just a bit weird